cve_bin_tool/cli.py

```diff
@@ -73,18 +73,29 @@ def main(argv=None):
         + "\n\nPlease disclose issues responsibly!",
         formatter_class=argparse.RawDescriptionHelpFormatter,
     )
+
+    nvd_database_group = parser.add_argument_group(
+        "NVD", "Arguments related to NVD Datbase and Cache Configuration"
```

Comment:
```suggestion
        "CVE data download", "Arguments related to NVD Database and Cache Configuration"
```

We've been kind of assuming that people know (or will look up) what the NVD is, but maybe we could use the heading to give them a hint here.